Cargo.toml

```diff
@@ -6,12 +6,18 @@ edition = "2018"
 description = "zvm goes zoom zoom"
 
 [[bin]]
-name = "class_info"
+name = "zvm-class-info"
 path = "src/class_info.rs"
 
 [[bin]]
 name = "zvm"
 path = "src/main.rs"
 
+[workspace]
+members = [
+    "zvm-vm",
```

Comment:
this wigs me out the the same way "atm machine" and "smh my head", i'd really like to preserve the vm just being `zvm`

let's do something kind how cranelift is structured, and call the root `zvm-tools`, keeping `class-info` here. i'd prefer not `zvm-class-info` because that name implies to me that zvm has an idea of classes independent to java classes. `class-info` definitely deserves a better name though, how's `bytecode-info` sound to you?

with the root crate being renamed `zvm-tools` the `zvm` `[[bin]]` section can move into the subcrate with the rest of `zvm`. which i think makes for a pretty good structure to expand on!

Reply:
Reordered with similar approach, slightly more name-oriented structure!